server/docker-configurations/users.yml

```diff
@@ -67,59 +67,59 @@ operatorfabric.users.default:
      name: Control Center FR North
      description: Control Center FR North
      parents : ["ENTITY_FR"]
-     roles: ["CARD_RECEIVER", "CARD_SENDER"]
+     roles: ["CARD_RECEIVER", "CARD_SENDER", "ACTIVITY_AREA"]
    - id: ENTITY2_FR
      name: Control Center FR South
      description: Control Center FR South
      parents : ["ENTITY_FR"]
-     roles: ["CARD_RECEIVER", "CARD_SENDER"]
+     roles: ["CARD_RECEIVER", "CARD_SENDER", "ACTIVITY_AREA"]
    - id: ENTITY3_FR
      name: Control Center FR East
      description: Control Center FR East
      parents : ["ENTITY_FR"]
-     roles: ["CARD_RECEIVER", "CARD_SENDER"]
+     roles: ["CARD_RECEIVER", "CARD_SENDER", "ACTIVITY_AREA"]
    - id: ENTITY4_FR
      name: Control Center FR West
      description: Control Center FR West
      parents : ["ENTITY_FR"]
-     roles: ["CARD_RECEIVER", "CARD_SENDER"]
+     roles: ["CARD_RECEIVER", "CARD_SENDER", "ACTIVITY_AREA"]
    - id: ENTITY_FR
      name: French Control Centers
      description: French Control Centers
-     roles: ["CARD_RECEIVER"]
+     roles: ["CARD_RECEIVER", "ACTIVITY_AREA_GROUP"]
    - id: IT_SUPERVISOR_ENTITY
      name: IT SUPERVISION CENTER 
      description: IT SUPERVISION CENTER
-     roles: ["CARD_RECEIVER", "CARD_SENDER"]
+     roles: ["CARD_RECEIVER", "CARD_SENDER", "ACTIVITY_AREA"]
    - id: ENTITY1_IT
      name: Control Center IT North
      description: Control Center IT North
      parents : ["ENTITY_IT"]
-     roles: ["CARD_RECEIVER", "CARD_SENDER"]
+     roles: ["CARD_RECEIVER", "CARD_SENDER", "ACTIVITY_AREA"]
    - id: ENTITY2_IT
      name: Control Center IT Center
      description: Control Center IT Center
      parents : ["ENTITY_IT"]
-     roles: ["CARD_RECEIVER", "CARD_SENDER"]
+     roles: ["CARD_RECEIVER", "CARD_SENDER", "ACTIVITY_AREA"]
    - id: ENTITY3_IT
      name: Control Center IT South
      description: Control Center IT South
-     parents : ["ENTITY_IT"]
+     parents : ["ENTITY_IT", "ACTIVITY_AREA"]
```

Comment:
It seems there was an error for this entity. It was already missing the roles before this PR. I think the correct declaration should be :
```
parents : ["ENTITY_IT"]
roles: ["CARD_RECEIVER", "CARD_SENDER", "ACTIVITY_AREA"]
```

Reply:
Done.